readers-frontend/src/components/HeaderNavbar.astro

```diff
@@ -0,0 +1,124 @@
+---
+import type { categories } from "@prisma/client";
+import prisma from "../prisma";
+import { Icon } from "astro-icon";
+
+interface Props {
+  active: string;
+}
+
+let menuCategories: categories[] = await prisma.categories.findMany({
+  where: {
+    categories_showMenu: true,
+    categories_nestUnder: null,
+    NOT: { categories_name: "muse" }, // Added manually because of image layout
+  },
+});
+
+// Add home as first item - possibly need to adjust values in the future
+menuCategories.unshift({
```

Comment:
No you need all elements because this is typescript. When you use `as` it's not a cast. You're telling the compiler to trust you on the type. We don't know which of those properties we'll need in the future so better not force the compiler to believe it is a `categories` item when it's in fact not compliant